[2025-04-19T15:47:18.016+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-19T15:47:18.095+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_etl_pipeline_seed.dbt_seed manual__2025-04-19T15:47:16.561941+00:00 [queued]>
[2025-04-19T15:47:18.110+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_etl_pipeline_seed.dbt_seed manual__2025-04-19T15:47:16.561941+00:00 [queued]>
[2025-04-19T15:47:18.112+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-19T15:47:18.132+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): dbt_seed> on 2025-04-19 15:47:16.561941+00:00
[2025-04-19T15:47:18.139+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=4563) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-19T15:47:18.141+0000] {standard_task_runner.py:72} INFO - Started process 4567 to run task
[2025-04-19T15:47:18.142+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'dbt_etl_pipeline_seed', 'dbt_seed', 'manual__2025-04-19T15:47:16.561941+00:00', '--job-id', '232', '--raw', '--subdir', 'DAGS_FOLDER/dbt_seed.py', '--cfg-path', '/tmp/tmp70zbwenk']
[2025-04-19T15:47:18.144+0000] {standard_task_runner.py:105} INFO - Job 232: Subtask dbt_seed
[2025-04-19T15:47:18.459+0000] {task_command.py:467} INFO - Running <TaskInstance: dbt_etl_pipeline_seed.dbt_seed manual__2025-04-19T15:47:16.561941+00:00 [running]> on host 27536697ee14
[2025-04-19T15:47:18.681+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='azizjon' AIRFLOW_CTX_DAG_ID='dbt_etl_pipeline_seed' AIRFLOW_CTX_TASK_ID='dbt_seed' AIRFLOW_CTX_EXECUTION_DATE='2025-04-19T15:47:16.561941+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-19T15:47:16.561941+00:00'
[2025-04-19T15:47:18.682+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-19T15:47:18.699+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-04-19T15:47:18.701+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', '\n        export PATH=/home/airflow/.local/bin:$PATH\n        cd /opt/airflow/commerce_dbt\n        dbt seed --profiles-dir .\n        ']
[2025-04-19T15:47:18.708+0000] {subprocess.py:99} INFO - Output:
[2025-04-19T15:47:20.899+0000] {subprocess.py:106} INFO - [0m15:47:20  Running with dbt=1.7.0
[2025-04-19T15:47:21.145+0000] {subprocess.py:106} INFO - [0m15:47:21  Registered adapter: postgres=1.7.0
[2025-04-19T15:47:21.582+0000] {subprocess.py:106} INFO - [0m15:47:21  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-04-19T15:47:21.583+0000] {subprocess.py:106} INFO - There are 1 unused configuration paths:
[2025-04-19T15:47:21.584+0000] {subprocess.py:106} INFO - - seeds.initial_dataset
[2025-04-19T15:47:21.634+0000] {subprocess.py:106} INFO - [0m15:47:21  Found 13 models, 1 snapshot, 6 seeds, 43 tests, 6 sources, 0 exposures, 0 metrics, 401 macros, 0 groups, 0 semantic models
[2025-04-19T15:47:21.638+0000] {subprocess.py:106} INFO - [0m15:47:21
[2025-04-19T15:47:22.012+0000] {subprocess.py:106} INFO - [0m15:47:22  Concurrency: 4 threads (target='dev')
[2025-04-19T15:47:22.012+0000] {subprocess.py:106} INFO - [0m15:47:22
[2025-04-19T15:47:22.024+0000] {subprocess.py:106} INFO - [0m15:47:22  1 of 6 START seed file commerce_raw.customers .................................. [RUN]
[2025-04-19T15:47:22.025+0000] {subprocess.py:106} INFO - [0m15:47:22  2 of 6 START seed file commerce_raw.order_items ................................ [RUN]
[2025-04-19T15:47:22.026+0000] {subprocess.py:106} INFO - [0m15:47:22  3 of 6 START seed file commerce_raw.orders ..................................... [RUN]
[2025-04-19T15:47:22.027+0000] {subprocess.py:106} INFO - [0m15:47:22  4 of 6 START seed file commerce_raw.product_categories ......................... [RUN]
[2025-04-19T15:47:23.089+0000] {subprocess.py:106} INFO - [0m15:47:23  4 of 6 OK loaded seed file commerce_raw.product_categories ..................... [[32mINSERT 4[0m in 1.02s]
[2025-04-19T15:47:23.140+0000] {subprocess.py:106} INFO - [0m15:47:23  5 of 6 START seed file commerce_raw.products ................................... [RUN]
[2025-04-19T15:47:23.817+0000] {subprocess.py:106} INFO - [0m15:47:23  5 of 6 OK loaded seed file commerce_raw.products ............................... [[32mINSERT 17[0m in 0.65s]
[2025-04-19T15:47:23.845+0000] {subprocess.py:106} INFO - [0m15:47:23  6 of 6 START seed file commerce_raw.stores ..................................... [RUN]
[2025-04-19T15:47:24.471+0000] {subprocess.py:106} INFO - [0m15:47:24  6 of 6 OK loaded seed file commerce_raw.stores ................................. [[32mINSERT 5[0m in 0.59s]
[2025-04-19T15:47:24.770+0000] {subprocess.py:106} INFO - [0m15:47:24  1 of 6 OK loaded seed file commerce_raw.customers .............................. [[32mINSERT 1000[0m in 2.74s]
[2025-04-19T15:47:34.178+0000] {subprocess.py:106} INFO - [0m15:47:34  3 of 6 OK loaded seed file commerce_raw.orders ................................. [[32mINSERT 7000[0m in 12.14s]
[2025-04-19T15:47:36.328+0000] {subprocess.py:106} INFO - [0m15:47:36  2 of 6 OK loaded seed file commerce_raw.order_items ............................ [[32mINSERT 10000[0m in 14.30s]
[2025-04-19T15:47:36.388+0000] {subprocess.py:106} INFO - [0m15:47:36
[2025-04-19T15:47:36.389+0000] {subprocess.py:106} INFO - [0m15:47:36  Finished running 6 seeds in 0 hours 0 minutes and 14.75 seconds (14.75s).
[2025-04-19T15:47:36.414+0000] {subprocess.py:106} INFO - [0m15:47:36
[2025-04-19T15:47:36.415+0000] {subprocess.py:106} INFO - [0m15:47:36  [32mCompleted successfully[0m
[2025-04-19T15:47:36.415+0000] {subprocess.py:106} INFO - [0m15:47:36
[2025-04-19T15:47:36.416+0000] {subprocess.py:106} INFO - [0m15:47:36  Done. PASS=6 WARN=0 ERROR=0 SKIP=0 TOTAL=6
[2025-04-19T15:47:37.601+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-04-19T15:47:37.729+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-19T15:47:37.731+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=dbt_etl_pipeline_seed, task_id=dbt_seed, run_id=manual__2025-04-19T15:47:16.561941+00:00, execution_date=20250419T154716, start_date=20250419T154718, end_date=20250419T154737
[2025-04-19T15:47:37.873+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-19T15:47:37.940+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-19T15:47:37.944+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
